src/Microsoft.Dnx.Runtime/ApplicationHostContext.cs

```diff
@@ -214,7 +221,19 @@ private static Result InitializeCore(ApplicationHostContext context)
             var lockFilePresent = context.LockFile != null;
             context.LockFile = null; // LockFile is no longer needed
 
-            return new Result(libraries, lockFilePresent, validLockFile, lockFileValidationMessage);
+            string minRequiredRuntime = null;
+            if (!context.SkipRuntimeMinVersionCheck)
+            {
+                var abstractions = libraries.FirstOrDefault(lib => lib.Identity.Name.Equals("Microsoft.Dnx.Runtime.Abstractions", StringComparison.Ordinal));
```

Comment:
@davidfowl is going to tell you to remove the Linq. No Linq allowed in the runtime core path :).